modules/apps/dispatch/dispatch-talend-web/src/main/java/com/liferay/dispatch/talend/web/internal/frontend/taglib/servlet/taglib/DispatchTalendScreenNavigationEntry.java

```diff
@@ -71,15 +71,15 @@ public String getLabel(Locale locale) {
 
 	@Override
 	public String getScreenNavigationKey() {
-		return DispatchConstants.SCREEN_NAVIGATION_KEY_DISPATCH_GENERAL;
+		return DispatchTalendScreenNavigationConstants.SCREEN_NAVIGATION_KEY_DISPATCH_GENERAL;
 	}
 
 	@Override
 	public boolean isVisible(User user, DispatchTrigger dispatchTrigger) {
 		if ((dispatchTrigger == null) ||
 			!Objects.equals(
 				dispatchTrigger.getDispatchTaskExecutorType(),
-				TalendDispatchTaskExecutor.TALEND) ||
+				DispatchTalendScreenNavigationConstants.CATEGORY_KEY_TALEND) ||
```

Comment:
Hey @MrJohn1911, here you should use the `TalendDispatchTaskExecutor.TALEND` since it is a checking not related to the `ScreenNavigation` logic. Thx